component/nats/BUCK

```diff
@@ -9,15 +9,16 @@ docker_image(
         "nats-server.conf": "component/nats",
     },
     build_args = {
-        "BASE_VERSION": "2.10.10",
+        "BASE_VERSION": "2.10.11",
     },
     run_docker_args = [
         "--publish",
         "4222:4222",
+        "--publish",
+        "8222:8222",
```

Comment:
Probably good to have it!

Reply:
Yeah, it will get used by `nats-top` which looks like a freakin' useful one. In prod it's a tossup whether that would be on or off, but great in dev either way